earth_enterprise/src/common/config/geConfigUtil.cpp

```diff
@@ -41,41 +41,14 @@ bool IsRedHat(void) {
 }
 
 uint GetNumCPUs(void) {
-  uint numcpus = 0;
-  std::ifstream in("/proc/cpuinfo");
-  if (in) {
-    std::string line;
-    while (std::getline(in, line)) {
-      if (StartsWith(line, "processor")) {
-        ++numcpus;
-      }
-    }
-  }
-
-  return std::max(uint(1), numcpus);
+  return std::max(uint(1), uint(sysconf(_SC_NPROCESSORS_ONLN)));
 }
 
+// want memory size in Kb
```

Comment:
I'm not sure what this comment means.  It looks like this function is trying to return the memory size in bytes.